src/components/Form/UploadDocumentForm.jsx

```diff
@@ -24,53 +23,73 @@ const UploadDocumentForm = () => {
   const { session } = useSession();
   const { state, dispatch } = useStatusNotification();
   const [docType, setDocType] = useState('');
+  const [verifyFile, setVerifyFile] = useState(false);
+  const [expiryDate, setExpiryDate] = useState('');
+  const [description, setDescription] = useState('');
+  const [file, setFile] = useState('');
+  const [inputKey, setInputKey] = useState('');
+  const { podUrl } = useContext(SignedInUserContext);
+  const { selectedUser } = useContext(SelectUserContext);
 
   const handleDocType = (event) => {
     setDocType(event.target.value);
   };
 
-  // Initalized state for file upload
-  const handleFileChange = (event) => {
-    if (event.target.files.length === 1) {
-      dispatch({ type: 'SET_FILE', payload: event.target.files[0] });
-    } else {
-      dispatch({ type: 'CLEAR_FILE' });
-    }
-  };
-
   // Custom useField hook for handling form inputs
-  const { clearValue: clearDescription, _type, ...description } = useField('textarea');
 
-  const clearInputFields = (event) => {
-    event.target.reset();
-    clearDescription();
-    dispatch({ type: 'CLEAR_FILE' });
-    dispatch({ type: 'CLEAR_VERIFY_FILE' });
+  const clearInputFields = () => {
+    setVerifyFile(false);
+    setDocType('');
+    setExpiryDate('');
+    setDescription('');
+    setFile(null);
+    setInputKey(!inputKey); // clears file by forcing re-render
     dispatch({ type: 'CLEAR_PROCESSING' });
   };
 
   // Event handler for form/document submission to Pod
-  const handleFormSubmit = makeHandleFormSubmission(
-    INTERACTION_TYPES.SELF,
-    state,
-    dispatch,
-    session,
-    clearInputFields
-  );
-
-  const handleDocumentUpload = async (event) => {
-    event.preventDefault();
-    dispatch({ type: 'SET_PROCESSING' });
+  const handleDocUpload = async (e) => {
+    e.preventDefault();
 
     if (!docType) {
-      runNotification('Search failed. Reason: No document type selected.', 5, state, dispatch);
+      runNotification('Upload failed. Reason: No document type selected.', 5, state, dispatch);
       setTimeout(() => {
         dispatch({ type: 'CLEAR_PROCESSING' });
       }, 3000);
       return;
     }
 
-    handleFormSubmit(event);
+    const fileDesc = {
+      name: file.name.split('.')[0],
+      type: docType,
+      date: expiryDate,
+      description
+    };
+    runNotification(`Uploading "${file.name}" to Solid...`, 3, state, dispatch);
+    const activePod = selectedUser.podUrl || podUrl;
+
+    try {
+      const doc = await createDocument(file, fileDesc, session, `${activePod}PASS/`);
+      if (verifyFile)
+        await signDocument(doc, session, `${activePod}PASS/${docType}/${file.name.split('.')[0]}/`);
```

Comment:
Just notice this now, it looks like an issue that was missed in PR #241.

It seems the url path to signature.ttl is still pointing to the old upload path of `/PASS/<docType>/` instead of the newer `/PASS/Documents/<docType>/` so signature.ttl was sent to the incorrect place.

![Screen Shot 2023-06-16 at 0 36 01](https://github.com/codeforpdx/PASS/assets/14917816/d4b2521c-5892-4c82-b92e-103bcdfb89b1)

Reply:
fixed